cli/flags.rs

```diff
@@ -256,138 +198,125 @@ pub fn set_flags(
   }
```

Comment:
`v8_set_flags` is called here, which makes it not a pure parsing function. Can that be moved elsewhere?

Reply:
So based on this [comment](https://github.com/denoland/deno/pull/2113#issuecomment-485135260) I'm gonna add `v8_help` and `v8_flags` fields to `DenoFlags` and call `v8_set_flags` in `main.rs::main()`

Reply:
Yes, sounds good.